languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/added.txt

```diff
@@ -8752,3 +8753,106 @@ aliançazinha	aliançazinha	NCFS000
 aliançazinhas	aliançazinha	NCFP000
 descompressor	descompressor	NCMS000
 descompressores	descompressor	NCMP000
+home	home	NCMS000
+homes	home	NCMP000
+PIN	PIN	NCMx000
+switch	switch	NCMS000
+Switch	switch	NCMS000
+SWITCH	switch	NCMS000
+switches	switch	NCMP000
+Switches	switch	NCMP000
+SWITCHES	switch	NCMP000
+prompt	prompt	NCMS000
+Prompt	prompt	NCMS000
+PROMPT	prompt	NCMS000
+prompts	prompt	NCMP000
+Prompts	prompt	NCMP000
+PROMPTS	prompt	NCMP000
+snippet	snippet	NCMS000
+Snippet	snippet	NCMS000
+SNIPPET	snippet	NCMS000
+webinares	webinar	NCMP000
+Webinares	webinar	NCMP000
+WEBINARES	webinar	NCMP000
+BANRISUL	Banrisul	NPMSO00
+déficit	déficit	NCMS000
+Déficit	déficit	NCMS000
+DÉFICIT	déficit	NCMS000
+déficits	déficit	NCMP000
+Déficits	déficit	NCMP000
+DÉFICITS	déficit	NCMP000
+Emmanuel	Emmanuel	NPMSS00
+Flavia	Flavia	NPFSS00	
+FLAVIA	Flavia	NPFSS00	
+Flavio	Flavio	NPMSS00	
+FLAVIO	Flavio	NPMSS00	
+gamificação	gamificação	NCFS000
+Gamificação	gamificação	NCFS000
+GAMIFICAÇÃO	gamificação	NCFS000
+Serasa	Serasa	NPMSO00
+SERASA	Serasa	NPMSO00
+ZAP	ZAP	NPMSO00
+##
+Izabela	Izabela	NPFSS00
+IZABELA	Izabela	NPFSS00
+Avon	Avon	NPFSO00
+AVON	Avon	NPFSO00
+Mark	Mark	NPMSS00
+MARK	Mark	NPMSS00
+Alison	Alison	NPMSS00
+ALISON	Alison	NPMSS00
```

Comment:
I feel like Alison is a unisex name? Best to tag it as NPCSS00

Reply:
tbh in Rio I think I only ever met male Alisons, but fair

Reply:
Yes, but as an international name, it also applies to women, so many people could be using it in pop culture references, etc. 😉